### PR TITLE
Feat/fe latest ratings movies grid view

### PR DESCRIPTION
**Pull request by na-talia**
# Description

Created grid view of movies with latest rating (with dummy data)

# Jira task link

https://hackyourfuture-dk.atlassian.net/jira/software/projects/LANDING/boards/6?selectedIssue=LANDING-29

# Screenshots (if applicable)

gridMovies

### CODE REVIEW

File: packages/client/src/components/LatestRatingMovies/LatestRatingMovies.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+export default function LatestRatingMovies() {
+  const [movies, setMovies] = useState([]);
+  useEffect(() => {
+    async function fetchMovies() {
+      const response = await fetch(`${apiURL()}/movies`);
+      const examples = await response.json();
+      setMovies(examples);
+    }
+    fetchMovies();
+  }, []);
```
Comment (magdazelena) [inline packages/client/src/components/LatestRatingMovies/LatestRatingMovies.jsx:17]: this is the happy scenario, perhaps you want to add loading and error scenario as well. Doesn't have to be a part of this PR :)

File: packages/client/src/components/LatestRatingMovies/LatestRatingMovies.jsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+          height: '2.76rem',
+          fontSize: '2.25rem',
+          fontWeight: 700,
+          color: '#FFFFFF',
+          textAlign: 'center',
+          marginBottom: '3rem',
+          marginTop: '3rem',
+          fontFamily: 'Inter',
+        }}
+      >
```
Comment (magdazelena) [inline packages/client/src/components/LatestRatingMovies/LatestRatingMovies.jsx:31]: As I mentioned in @maccrin PR, you guys might want to discuss in teams making some sort of component library internally in the project so you don't have to style everything all the time if the components repeat, like this header for example :) It is always easier later to change things as you have one source and maintain consitency, not to mention readability :)

File: packages/client/src/components/LatestRatingMovies/LatestRatingMovies.jsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        LATEST RATINGS
+      </Typography>
+      <Grid
+        sx={{
+          width: '78rem',
+          maxHeight: 'fit-content',
+          marginLeft: '5rem',
+          // alignItems: 'center'
```
Comment (magdazelena) [inline packages/client/src/components/LatestRatingMovies/LatestRatingMovies.jsx:39]: some forgotten comment here :)

**Review by magdazelena — APPROVED** [review]
couple small comments, nothing blocking :)

File: packages/client/src/components/LatestRatingMovies/LatestRatingMovies.jsx (hunk 1 above)
Comment (maccrin) [reply packages/client/src/components/LatestRatingMovies/LatestRatingMovies.jsx:17]: Hi 
Thanks for review,I made this with dummy data as back end api still is not working :).

Comment (na-talia) [reply packages/client/src/components/LatestRatingMovies/LatestRatingMovies.jsx:17]: Ok, I will add it :)

File: packages/client/src/components/LatestRatingMovies/LatestRatingMovies.jsx (hunk 2 above)
Comment (na-talia) [reply packages/client/src/components/LatestRatingMovies/LatestRatingMovies.jsx:31]: Yes, we should definitely do it. Thank you!

**Comment by na-talia** [comment]
> couple small comments, nothing blocking :)

Thank you so much for your comments! :)

**Pull request merged**